tests/functional-ui/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "targets": {
+    "lint": {},
+    "check": {},
+    "functional-ui": {
+      "cache": false,
+      "persistent": false
+    },
+    "functional-ui:golang": {
+      "cache": false,
```

Comment:
Why can this not be cached?

Reply:
We anyway don't run this with Nx, yet. This is fixed with https://github.com/zitadel/zitadel/pull/10744